authors/apps/authentication/views.py

```diff
@@ -18,16 +28,58 @@ class RegistrationAPIView(APIView):
 
     def post(self, request):
         user = request.data.get('user', {})
-
         # The create serializer, validate serializer, save serializer pattern
         # below is common and you will see it a lot throughout this course and
         # your own work later on. Get familiar with it.
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
-        serializer.save()
-        
+        user = serializer.save()
+        user.is_active = False
+        user.save()
+        current_site = get_current_site(request)
+        response_data = self.send_email_notification(current_site, user)
+        response_data.update(serializer.data)
+        return Response(response_data, status=status.HTTP_201_CREATED)
+
+    def send_email_notification(self, current_site, user):
+        domain = f'http://{current_site.domain}'
+        token = default_token_generator.make_token(user)
+        uid = urlsafe_base64_encode(force_bytes(user.username)).decode('utf-8')
+        route = reverse('authentication:confirm', args=[token, uid])
+        url = f'{domain}{route}'
+        subject = "Activate your Author's Haven Account."
+        message = f'You have successfully registered an AH account. \n Click \
+                    Link Below To Activate Your Account\n{url}'
```

Comment:
![image](https://user-images.githubusercontent.com/26769886/51182138-402ce080-18de-11e9-8599-a437c0ddbd5b.png)

Fix this bug, there's too much space appearing in the message.

Reply:
Thanks for pointing that out I have now fixed the issue